Core.Collectors/IO/StorageAccountHelper.cs

```diff
@@ -16,7 +16,7 @@ public static CloudStorageAccount GetStorageAccount(string storageConnectionEnvi
             return CloudStorageAccount.Parse(stagingBlobConnectionString);
         }
 
-        public async static Task<CloudStorageAccount> GetStorageAccountUsingMsi(string storageAccountNameEnvironmentVariable = "StorageAccountName", bool isQueue = false)
+        public async static Task<CloudStorageAccount> GetStorageAccountUsingMsiAsync(string storageAccountNameEnvironmentVariable = "StorageAccountName", bool isQueue = false)
```

Comment:
>string storageAccountNameEnvironmentVariable = "StorageAccountName" [](http://example.com/codeflow?start=85&length=67)

Any reason you are not removing it from here?

Reply:
missed this